src/structify/resources/structure.py

```diff
@@ -259,6 +259,45 @@ def run_async(
             ),
             cast_to=object,
         )
+    
+    def run(
+        self,
+        *,
+        dataset_name: str,
+        text: structure_run_async_params.Variant0Text,
+        custom_instruction: Optional[str] | NotGiven = NOT_GIVEN,
+        # Use the following arguments if you need to pass additional parameters to the API that aren't available via kwargs.
+        # The extra values given here take precedence over values defined on the client or passed to this method.
+        extra_headers: Headers | None = None,
+        extra_query: Query | None = None,
+        extra_body: Body | None = None,
+        timeout: float | httpx.Timeout | None | NotGiven = NOT_GIVEN,
```

Comment:
Do we need these? They're sorta awkward as part of the API

Reply:
Removed with workaround.